src/main/java/hw3/DifferentElementsPage.java

```diff
@@ -0,0 +1,90 @@
+package hw3;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class DifferentElementsPage extends AbstractPage {
+
+    @FindBy(xpath = "//input[@type = 'checkbox']")
+    private List<WebElement> checkboxes;
+
+    @FindBy(xpath = "//input[@type = 'checkbox'][1]")
+    private WebElement checkboxWater;
+
+    @FindBy(xpath = "//input[@type = 'checkbox'][3]")
+    private WebElement checkboxWind;
+
+    @FindBy(xpath = "//input[@type = 'radio']")
+    private List<WebElement> radios;
+
+    @FindBy(xpath = "//input[@type = 'radio'][3]")
+    private WebElement radioSelen;
```

Comment:
Please try avoid using indexes of the tags in the xpath locator. These locators very unstable